src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  sides: number[];
+
+  constructor(color: string, a: number, b: number, c: number) {
```

Comment:
You can rewrite your constructor to make it easier to read
```suggestion
  constructor(
    color: string, 
    a: number, 
    b: number, 
    c: number,
  ) {
```